lib/posix-netlink/include/uk/netlink_protocol.h

```diff
@@ -0,0 +1,133 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
According to our conventions this should be `include/uk/netlink.h`. If it's only intended for drivers though, it could be alternatively `include/uk/netlink/driver.h`.